fhir-search/src/test/java/com/ibm/fhir/search/location/NearLocationHandlerBoundingRadiusTest.java

```diff
@@ -21,12 +21,13 @@
 import com.ibm.fhir.search.location.bounding.Bounding;
 import com.ibm.fhir.search.location.bounding.BoundingRadius;
 import com.ibm.fhir.search.location.bounding.BoundingType;
+import com.ibm.fhir.search.test.BaseSearchTest;
 import com.ibm.fhir.search.util.SearchUtil;
 
 /**
  * Test the BoundingRadius with NearLocationHandler
  */
-public class NearLocationHandlerBoundingRadiusTest {
```

Comment:
Why the extends?

Reply:
Depending on the order the tests are run, sometimes FHIRRequestContext was null, which causes a NPE down within the call to parseQueryParameters(...), so I extended BaseSearchTest to ensure it was set.